src/Microsoft.ManagedZlib/tests/MyClassTests.cs

```diff
@@ -0,0 +1,339 @@
+    //    LocalMemoryStream compressedStream = await LocalMemoryStream.readAppFileAsync(CompressedTestFile(testFile));
+    //    using Stream decompressor = CreateStream(compressedStream, CompressionMode.Decompress);
+    //    var decompressorOutput = new MemoryStream();
+
+    //    int _bufferSize = 1024;
+    //    var bytes = new byte[_bufferSize];
+    //    bool finished = false;
+    //    int retCount;
+    //    while (!finished)
+    //    {
+    //        retCount = decompressor.Read(bytes, 0, _bufferSize);
+
+    //        if (retCount != 0)
+    //            decompressorOutput.Write(bytes, 0, retCount);
+    //        else
+    //            finished = true;
+    //    }
+    //    decompressor.Dispose();
+    //    decompressorOutput.Position = 0;
+    //    uncompressedStream.Position = 0;
+
+    //    byte[] uncompressedStreamBytes = uncompressedStream.ToArray();
+    //    byte[] decompressorOutputBytes = decompressorOutput.ToArray();
+
+    //    Assert.Equal(uncompressedStreamBytes.Length, decompressorOutputBytes.Length);
+    //    for (int i = 0; i < uncompressedStreamBytes.Length; i++)
+    //    {
+    //        Assert.Equal(uncompressedStreamBytes[i], decompressorOutputBytes[i]);
+    //    }
+    //}
+    [Fact]
+    public void Test1()
+    {
+        using MemoryStream original = new MemoryStream();
+
+        byte[] originalBytes = new byte[] { 0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41,
+                                            0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41,
+                                            0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41,
+                                            0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41,
+                                            0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41,
+                                            0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41,
+                                            0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41,
+                                            0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41, 0x4C, 0x4C, 0x4F, 0x52, 0x41};
+        byte[] compressedBytes = new byte[10];
+        byte[] finalBytes = new byte[10];
+        original.Write(originalBytes);
+
+        MemoryStream originalData = new(); 
+        originalData.Write(originalBytes);
+        originalData.Position = 0;
+
+        MemoryStream compressedDestination = new();
+        using (System.IO.Compression.DeflateStream compressor = new System.IO.Compression.DeflateStream(compressedDestination, System.IO.Compression.CompressionMode.Compress, leaveOpen: true))
```

Comment:
This is wrong, we shouldn't be testing anything from System.IO.Compression. What this code currently doing is testing our official implementation, not yours.

```suggestion
        using (DeflateStream compressor = new DeflateStream(compressedDestination, CompressionMode.Compress, leaveOpen: true))
```